### PR TITLE
tclap@1.2.5

### PR DESCRIPTION
**Pull request by mmorel-35**
Release: https://github.com/mirror/tclap/releases/tag/v1.2.5

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (tclap) have been updated in this PR. Please review the changes.

File: modules/tclap/1.2.5/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module(
+    name = "tclap",
+    version = "1.2.5",
+    repo_name = "com_github_mirror_tclap"
```
Comment (meteorcloudy) [inline modules/tclap/1.2.5/MODULE.bazel:4]: Why do we need `repo_name = "com_github_mirror_tclap"`

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

**Review by meteorcloudy — DISMISSED** [review]
Thanks!

File: modules/tclap/metadata.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/mirror/tclap",
+    "maintainers": [
+        {
+            "github": "DAarno",
```
Comment (meteorcloudy) [inline modules/tclap/metadata.json:5]: Wait, will @DAarno actually help maintain this module? This should be the module maintainer for BCR, it doesn't have to be the maintainer of the original project.

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**